src/DynamoCore/Linting/Interfaces/IRuleEvaluationResult.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Linting.Interfaces
+{
+    public interface IRuleEvaluationResult
```

Comment:
I'm wondering if we really need this as an interface. I'm thinking to be more restrictive and maybe only use a simple structure which along with RuleEvaluationStatusEnum and SeverityCodesEnum should cover what we need in terms of evaluation results.